.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,25 @@
+name: "Feature request"
+description: Suggest an idea for this project
+title: "[FR]: "
+labels: ["enhancement", "Needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for suggesting an idea to improve Turbinia.
+        
+        Please fill in as much of the following form as you're able.
+  - type: textarea
+    attributes:
+      label: What is the problem this feature will solve?
```

Comment:
Not all feature requests solve "problems" per se, so this could maybe refer to potential "improvements" as well, ie. something like `What would this feature improve or what problem would it solve`

Reply:
Ack that makes sense, good callout. I went ahead and updated to something like the above ^^